src/doc/python_scripting/functions.rst

```diff
@@ -1799,37 +1747,14 @@ expression : string
     The expression definition as a string.
 
 return type : CLI_return_t
-    The DefineExpression functions return 1 on success and 0 on failure.
+    The DefineSpeciesExpression function returns 1 on success and 0 on failure.
```

Comment:
Same remark as for material expressions

Reply:
I started a discussion on this. Perhaps we should provide a way to create a species variable.